### PR TITLE
fix: Fix issue where Notion subpages did not render correctly

### PR DESCRIPTION
**Pull request by MichaelZhao21**
Notion subpages were redirecting to `/[pageId]` which is bad. Now any linked pages will be redirected to `hackerpacks/[pageId]`, which has a catch-all path to render the Notion page with the passed-in pageId.

### CODE REVIEW

**Comment by MichaelZhao21** [comment]
how do u mark PRs as draft

**Comment by nam-t24** [comment]
> how do u mark PRs as draft

Convert to draft under Reviewers

**Comment by MichaelZhao21** [comment]
pls review :3

File: pages/hackerpacks/index.tsx [hunk 1]
```diff
@@ -160,7 +160,11 @@ export default function HackerPack(props: { content: any }) {
       {/* Generate main content based on mainContent setting [notion, markdown, html] */}
       <section id="mainContent" className={`px-6 py-3 relative w-full ${adjustedWidth}`}>
         {hackerpackSettings.mainContent === 'notion' && (
-          <NotionRenderer recordMap={props.content} darkMode={false} />
+          <NotionRenderer
+            recordMap={props.content}
+            darkMode={false}
```
Comment (ZzRanger) [inline pages/hackerpacks/index.tsx:165]: darkMode >> lightMode

**Review by thetechie7 — APPROVED** [review]

**Review by abdullah50053 — APPROVED** [review]

**Review by nam-t24 — APPROVED** [review]

**Pull request merged**